src/ui/public/courier/fetch/strategy/__tests__/search.js

```diff
@@ -3,23 +3,33 @@ import expect from 'expect.js';
 import ngMock from 'ng_mock';
 
 import SearchStrategyProvider from '../search';
+import StubIndexPatternProv from 'test_utils/stub_index_pattern';
 
 describe('ui/courier/fetch/strategy/search', () => {
-  let $rootScope;
+  require('test_utils/no_digest_promises').activateForSuite();
+
   let search;
   let reqsFetchParams;
+  let IndexPattern;
 
   beforeEach(ngMock.module('kibana'));
 
-  beforeEach(ngMock.inject((Private, $injector) => {
-    $rootScope = $injector.get('$rootScope');
+  beforeEach(ngMock.inject((Private) => {
     search = Private(SearchStrategyProvider);
+    IndexPattern = Private(StubIndexPatternProv);
+
     reqsFetchParams = [
       {
         index: ['logstash-123'],
         type: 'blah',
         search_type: 'blah2',
         body: { foo: 'bar', $foo: 'bar' }
+      },
+      {
+        index: ['logstash-123'],
+        type: 'hi',
+        search_type: 'bye',
+        body: { foo: 'bar', $foo: 'bar' }
```

Comment:
Initially it was because I thought adding a second one would catch the issue.  Then I just left it in because two is better than one... but I suppose you are right, no point if I'm not adjusting the checks as well.  Removed!